tests/src/Functional/AlertBannerBlockTest.php

```diff
@@ -32,21 +33,44 @@ class AlertBannerBlockTest extends BrowserTestBase {
   protected $adminUser;
 
   /**
-   * {@inheritdoc}
+   * Place the alert block.
+   *
+   * Utility to place the alert banner block so we can assert
+   * against it's output.
```

Comment:
against it's output -> against its output

Reply:
Fixed typo in new commit